pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -315,6 +322,21 @@ def exist_in_previous_releases(vcf_entry):
     return False
 
 
+def choose_consensus_entry(consensus_entries, donor_unique_id):
+    # It is very naive now to append all the entries for the same variant type.(snv_mnv/indel/sv/cnv)
+    # The assumption: there is no duplicated uploading for the same aliquot of the same variant type
```

Comment:
That's good for now. Even if we need to do some basic checks, for example, the coming aliquot has not been processed before, it should be doable. But let's not worry about it now.